pyproject.toml

```diff
@@ -69,6 +69,10 @@ bigdl-cpu = [
     "bigdl-llm[all]"
 ]
 
+vllm = [
+    "vllm>=0.2.6"
+]
+
```

Comment:
It seems that you install vllm in a script, not by pip extra.

Reply:
Yes, we need to fix this as currently vllm-cpu installation is not perfect yet. Just leave it as a placeholder. I will remove this.